### DISCUSSION TITLE
chat-with-your-data-solution-accelerator/blob/main/media/

### DISCUSSION THREAD

**Discussion by joeseal** [Azure-Samples/chat-with-your-data-solution-accelerator#231] [Q&A] [answered]
Is there a way to view media png files? Noe of the media files seem to work.

https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/blob/main/media

**Comment by ross-p-smith** [comment, marked as answer]
Hi there, yes there was a problem with media images back in January - this has been fixed now. Please raise a bug if this is not working.
